### DISCUSSION TITLE
Incompatibility with version 25.2.27127.0?

### DISCUSSION THREAD

**Discussion by ssouthurst** [microsoft/ALAppExtensions#27680] [Q&A] [answered]
Has anyone else received a notification that their extension will be incompatible with v25.2 (apparently the next release)?  I received one and I can't find any information on what it claims will be the breaking changes.  What's more the breaking changes don't appear to be in the latest code for this Repo.  And they don't make sense (Getting rid of a number of "No" fields etc.).  These are the breaking changes listed:

`Error AVS0005: A package with publisher 'Microsoft', name 'System', and a version compatible with '25.2.0.0' could not be found. VendorAPI.al (479,23) - Error AL0185: Table '2000000041' is missing CustomerAPI.al (552,31) - Error AL0118: The name '"No."' does not exist in the current context. ItemCategoriesAPI.al (140,31) - Error AL0118: The name '"No."' does not exist in the current context. MergeDuplicatesAPI.al (381,41) - Error AL0118: The name '"Related Table ID"' does not exist in the current context. MergeDuplicatesAPI.al (382,41) - Error AL0118: The name '"Related Field No."' does not exist in the current context. PaymentMethodAPI.al (80,31) - Error AL0118: The name '"No."' does not exist in the current context. VendorAPI.al (524,31) - Error AL0118: The name '"No."' does not exist in the current context. CustomerAPI.al (503,23) - Error AL0185: Table '2000000041' is missing InvoiceLineAPI.al (504,27) - Error AL0185: Table '2000000041' is missing ItemAPI.al (105,23) - Error AL0185: Table '2000000041' is missing PurchaseInvoiceAggregatorRkn.al (606,23) - Error AL0185: Table 'Field' is missing ItemCategoriesAPI.al (131,23) - Error AL0185: Table '2000000041' is missing MergeDuplicatesAPI.al (374,33) - Error AL0185: Table 'Table Relations Metadata' is missing PaymentMethodAPI.al (71,23) - Error AL0185: Table '2000000041' is missing PurchaseInvoiceLinesAPI.al (397,27) - Error AL0185: Table '2000000041' is missing`

Anyone have any insights on this?

**Comment by NKarolak** [comment, marked as answer]
You can ignore this temporary bug:
https://www.linkedin.com/posts/stefano-demiliani_msdyn365bc-activity-7266512057569583104-zzRZ

**Reply by ssouthurst** [reply]
Legend - thanks!
